components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
This seems to be blown out of proportion :)

#1 reason for not sending to server is 10 lines of js code vs 100+ lines of js+velocity+java code, since 10 lines of regexp in JS would have to be replaced with 1) at least as many lines to call Ajax, and asynchronously wait for a reply 2) new script service or new (unrelated to anything else) entry point for an existing one 3) new java classes to process SVG + time figuring out how Java libs handle SVG structure (I presume regular html/xml parsers would do, but never tried). Each js regexp line would also likely have to be replaced with more than one java line on average as well + boilerplate